script.js

```diff
@@ -0,0 +1,23 @@
+const menuIcon = document.getElementById('menu-icon');
+const mobileMenu = document.getElementById('mobile-menu');
+const closeMenuIcon = document.getElementById('close-menu');
+
+menuIcon.addEventListener('click', () => {
+  if (mobileMenu.style.display === 'none') {
+    mobileMenu.style.display = 'block';
+  } else {
+    mobileMenu.style.display = 'none';
+  }
+});
+
+closeMenuIcon.addEventListener('click', () => {
+  if (mobileMenu.style.display === 'block') {
+    mobileMenu.style.display = 'none';
+  }
+});
+
+mobileMenu.addEventListener('click', () => {
+  if (mobileMenu.style.display === 'block') {
+    mobileMenu.style.display = 'none';
+  }
+});
```

Comment:
You have done a nice job implementing the functionalities of the mobile menu😉 but there are still some things missing😉

- [x] Your mobile menu looks a bit different from the design😉 kindly see the design below and make the necessary comparison😉

your mobile menu
<img width="292" alt="Your_mobile_menu" src="https://user-images.githubusercontent.com/64643995/218188317-2cd3db82-20b1-40f2-8c7f-dde68c1cdadf.png">

Design😉
<img width="218" alt="mobile_menuTenplate2design" src="https://user-images.githubusercontent.com/64643995/218188481-ed61091b-1773-449d-a247-304df3d8bc2c.png">

when a user clicks the hamburger bar, the mobile menu pops up and takes the entire width of the screen, as given in the[ design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=gVAewYxoJijJg4Z4-0)😉 and the mobile menu remains fixed on the screen till the close button is clicked😉😉 currently, your mobile menu isn't fixed and it is not occupying the full width as stated in the given design🤗🤗 kindly make the necessary changes as it is a[ requirement](https://github.com/egichuhis/Edwin-Portfolio/pull/12/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3R1-R23) that we stick to the given design🤗🤗

- [x]  Also it is a[ requirement](https://github.com/egichuhis/Edwin-Portfolio/pull/12/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3R1-R23) that when we move from mobile to desktop, the mobile menu disappears.😉 currently, your mobile menu remains even when we move from mobile to desktop🤗🤗
<img width="929" alt="mobime_buggg" src="https://user-images.githubusercontent.com/64643995/218189386-3a2d6856-8cf9-49c9-8435-11a73bda6c30.png">
kindly fix these issues as they could lead to a bad user experience😉

Reply:
Thanks for noting this 😉